sanic/sanic.py

```diff
@@ -47,9 +47,8 @@ def response(handler):
     # Decorator
     def exception(self, *exceptions):
         """
-        Decorates a function to be registered as a route
-        :param uri: path of the URL
-        :param methods: list or tuple of methods allowed
+        Decorates a function to be registered as a handler for exceptions
+        :param *exceptions: exceptions
```

Comment:
In this method, exceptions parameter is tuple.
I think that the line should be changed as following.

```
:param *exceptions: tuple of exceptions
```